opendebates/templates/opendebates/list_ideas.html

```diff
@@ -81,6 +81,11 @@
           <option value="+votes" {% if sort == "+votes" %}selected{% endif %}>
             {% blocktrans %}Fewest Votes{% endblocktrans %}
           </option>
+          {% if PREVIOUS_DEBATE_TIME %}
```

Comment:
PREVIOUS_DEBATE_TIME is set by context processor and ensures the mode.previous_debate_time value is actually in the past...is that why this comment didn't result in a change?